walletnode/services/artworkregister/node/node.go

```diff
@@ -49,7 +50,9 @@ func (node *Node) Connect(ctx context.Context, timeout time.Duration) error {
 
 	connCtx, connCancel := context.WithTimeout(ctx, timeout)
 	defer connCancel()
-
+	if node.Client == nil {
+		fmt.Println("im ni")
```

Comment:
should return error.

Reply:
It was a typo. removed. Thanks for the catch